openedx/features/partners/migrations/0005_auto_20200320_0557.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's merge this migration with the previous one, and create a single migration.